RealmSwift/Tests/RealmTests.swift

```diff
@@ -1903,3 +1903,96 @@ extension CancellationError: Equatable {
         true
     }
 }
+
+// Helper
+extension LogLevel {
+    var logLevel: String {
+        switch self {
+        case .off:
+            return "Off"
+        case .fatal:
+            return "Fatal"
+        case .error:
+            return "Error"
+        case .warn:
+            return "Warn"
+        case .info:
+            return "Info"
+        case .detail:
+            return "Details"
+        case .debug:
+            return "Debug"
+        case .trace:
+            return "Trace"
+        case .all:
+            return "All"
+        default:
+            return "unknown"
+        }
+    }
+}
+
+@available(macOS 12.0, watchOS 8.0, iOS 15.0, tvOS 15.0, macCatalyst 15.0, *)
+class LoggerTests: TestCase {
```

Comment:
These test cases which change the shared logger should have a tearDown function to restore it to a normal state afterwards.